File: Abstraer/src/abstraer/Persona.java


/**
 * 
 */
package abstraer;
//Super Clase
/**
 *
 * @author Manuela Garcia Monsalve 
 */
public abstract class Persona {
	String nombre;
    String apellido;
    String fechaNac;
    int edad;
    
    
    /*
     * constructor de la clase persona
     * este constructor cuenta con los siguientes parametros:
     * nombre, apellido, edad, fecha de nacimiento
     */
    public Persona(String nom, String ape,  int ed, String fechaNa){
        this.nombre=nom;
        this.apellido=ape;
        this.edad=ed;
       this.fechaNac=fechaNa;
   
    }
    
    
    //setters y getters 
    // get de nombre 
	public String getNombre() {
		return nombre;
	}
	//set de nombre
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	//get de apellido
	public String getApellido() {
		return apellido;
	}
	// set de apellido
	public void setApellido(String apellido) {
		this.apellido = apellido;
	}
	//get de fecha de nacimiento 
	public String getFechaNac() {
		return fechaNac;
	}
	//set de fecha de nacimiento 
	public void setFechaNac(String fechaNac) {
		this.fechaNac = fechaNac;
	}
	//get de edad
	public int getEdad() {
		return edad;
	}
	// set de edad
	public void setEdad(int edad) {
		this.edad = edad;
	}

	
    
    abstract public void verDatos();
 
}

File: Abstraccion/Persona.py
""""en este programa se pretende utlizar la herencia
AUTOR Manuela Garcia Monsalve
fecha: 23 mayo 2018"""




class persona: # clase persona
    def __init__(self, nombre, apellido, edad, fechaN): # se crea el constructor que recibe los parametros.
        """"entre los parametros estan:
        cedula
        nombre
        apellido
        edad
         fecha nacimiento """
        self.nombre=nombre
        self.apellido=apellido
        self.edad=edad
        self.fechaN=fechaN

    #get de nombre
    def getNombre(self):
        return self.nombre
    # set de nombre
    def setNombre(self, nombre):
        self.nombre= nombre
    #get de apellido
    def getApellido(self):
        return  self.apellido
    #set de apllido
    def setApellido(self, apellido):
        self.apellido=apellido
    # get de edad
    def getEdad(self):
        return  self.edad
    #set de edad
    def setEdad(self, edad):
        self.edad=edad
    #get de fecha de nacimiento
    def getFechaN(self):
        return  self.fechaN
    #set de fecha de nacimiento
    def serFechaN(self, fechaN):
        self.fechaN= fechaN

    def verDatos(self):

        pass


